generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -31,27 +590,3421 @@ enum Boolean {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchNumberOfUsersResponse {
+  int32 status = 1;
+  int32 number_of_users = 2;
+}
+
+message OFDMModAccFetchNumberofSymbolsUsedRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchNumberofSymbolsUsedResponse {
+  int32 status = 1;
+  int32 number_of_symbols_used = 2;
+}
+
+message OFDMModAccFetchPEAveragePowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPEAveragePowerResponse {
+  int32 status = 1;
+  double pe_average_power_mean = 2;
+}
+
+message OFDMModAccFetchPEDurationRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPEDurationResponse {
+  int32 status = 1;
+  double pe_duration = 2;
+}
+
+message OFDMModAccFetchPEPeakPowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPEPeakPowerResponse {
+  int32 status = 1;
+  double pe_peak_power_maximum = 2;
+}
+
+message OFDMModAccFetchPPDUAveragePowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPPDUAveragePowerResponse {
+  int32 status = 1;
+  double ppdu_average_power_mean = 2;
+}
+
+message OFDMModAccFetchPPDUPeakPowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPPDUPeakPowerResponse {
+  int32 status = 1;
+  double ppdu_peak_power_maximum = 2;
+}
+
+message OFDMModAccFetchPPDUTypeRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPPDUTypeResponse {
+  int32 status = 1;
+  OfdmPpduType ppdu_type = 2;
+  int32 ppdu_type_raw = 3;
+}
+
+message OFDMModAccFetchPSDUCRCStatusRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPSDUCRCStatusResponse {
+  int32 status = 1;
+  OfdmModAccPsduCrcStatus psducrc_status = 2;
+  int32 psducrc_status_raw = 3;
+}
+
+message OFDMModAccFetchPilotConstellationTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPilotConstellationTraceResponse {
+  int32 status = 1;
+  repeated nidevice_grpc.NIComplexNumberF32 pilot_constellation = 2;
+  int32 actual_array_size = 3;
+}
+
+message OFDMModAccFetchPreambleAveragePowers80211acRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchPreambleAveragePowers80211acResponse {
+  int32 status = 1;
+  double vhtsiga_average_power_mean = 2;
```

Comment:
there are several of these that I think need to be split. `vht_siga`, `he_sigb`, etc.